frontend/admin-client/src/app/bookings/booking-list/booking-list.component.ts

```diff
@@ -20,7 +20,14 @@ export class BookingListComponent implements OnInit {
     this.loading = true;
     this.bookingService.getAllBookings(environment.bookingApiUrl)
       .subscribe(bookings => {
-        console.log(bookings);
+        console.log('all bookings:', bookings);
+        bookings.forEach(booking => {
```

Comment:
you could use .map here instead to make it clearer that you are mutating bookings/flights